pkg/agent/flowexporter/connections/conntrack.go

```diff
@@ -46,11 +46,11 @@ func filterAntreaConns(conns []*flowexporter.Connection, nodeConfig *config.Node
 
 		// Consider Pod-to-Pod, Pod-To-Service and Pod-To-External flows.
 		if srcIP.Equal(nodeConfig.GatewayConfig.IPv4) || dstIP.Equal(nodeConfig.GatewayConfig.IPv4) {
-			klog.V(4).Infof("Detected flow for which one of the endpoint is host gateway %s :%+v", nodeConfig.GatewayConfig.IPv4.String(), conn)
```

Comment:
It's a pretty small change here but it would be best not to mix up such changes with changes to the test infra

Reply:
I was also doubtful whether to add it or not. I removed it; will do it in a separate PR.